Gemfile

```diff
@@ -235,7 +235,7 @@ gem "factory_bot_rails", "~> 6.4.4", require: false
 gem "turbo_power", "~> 0.7.0"
 gem "turbo-rails", "~> 2.0.0"
 
-gem "httpx"
+gem "httpx", "~> 1.3.4"
```

Comment:
IMO we should add a comment here, why we restrict the version and (probably much harder), we should try to create an upstream issue with reproduction instructions.

Did you get any idea how/why the newer version of HTTPX does that? Because it's working fine for most of our tests, so it surprises me that for one test it's showing this behaviour...

Reply:
No, I have no idea why yet. But it seems to be the case for all failures when VCR or WebMock stub can not be found. Then there is an infinite hang happening. I'll add a clarification comment.

Reply:
The new comment is good, because it explains what to look out for to update.

In case you write/find an issue in their issue tracker, this would also be a great place to cross link it.